apps/gallery/js/ImageEditor.js

```diff
@@ -699,9 +697,6 @@ function saveEditedImage() {
  *   cropImage()
  *   undoCrop()
  *
```

Comment:
Looking at Effects variable Line 1883, matrices are now  3x4, we should update comment above to reflect that.